.circleci/config.yml

```diff
@@ -28,15 +28,15 @@ workflows:
 jobs:
   package_schema:
     docker:
-      - image: rust:1.43.1
+      - image: rust:1.44.1
```

Comment:
Just curious, can the version string be put into a variable or something, to set it only in one place?

Reply:
I don't know a way to do it, but did never search for one. Search/replace (possible project wide) does the job for me.